src/data/staking-products.json

```
@@ -403,6 +403,49 @@
         "eventName": "Clicked stakefish go to link"
       }
     },
+    {
+      "name": "P2P.org",
+      "imageName": "P2P",
+      "hue": 227,
+      "launchDate": "2023-01-10",
+      "url": "https://p2p.org/networks/ethereum/",
+      "audits": [
+        {
+          "name": "Mixbytes (v1)",
+          "url": "https://github.com/mixbytes/audits_public/blob/79cd8153bfb6098227b43dfba8d061881a94a248/P2P.org/ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20(v.1)/P2P.org_ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20Smart%20Contracts%20Audit%20Report.pdf"
+        },
+        {
+          "name": "Mixbytes (v2)",
+          "url": "https://github.com/mixbytes/audits_public/blob/79cd8153bfb6098227b43dfba8d061881a94a248/P2P.org/ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20(v.2)/ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20Security%20Audit%20Report%20(v.2).pdf"
+        }
+      ],
+      "minEth": 32,
+      "additionalStake": null,
+      "additionalStakeUnit": null,
+      "monthlyFee": 5,
+      "monthlyFeeUnit": "%",
+      "isFoss": false,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "pctMajorityExecutionClient": 50,
+      "pctMajorityConsensusClient": 30,
+      "isSelfCustody": true,
```

From current "Self Custody" definition: 

> User maintains custody of any validator credentials, including signing and withdrawal keys

and your note:

> Client have no access to the signing keys

This wouldn't qualify currently, though I did raise this as an issue to potentially alter the definition here to account for the security of having withdrawal credentials set to the users own account: #11405

```suggestion
      "isSelfCustody": false,
```

I would encourage you to comment about this on that issue in the meantime 🙏

---

True, regarding your definition we are not self custody at the moment. We just put different value in the "custody" 
 ownership of ETH. So we position ourselves self custody because we don't have access to client's ETH's. We still have plans for the future to make convenient yet self custody in your definition.
 
 So if the definition is so, I guess we have no options.

---

Fully understand... this has been getting raised a bit more lately which was why I raised that issue. Again, if you wouldn't mind expressing some thoughts on that PR it would certainly help reassure the direction and help it gain some traction.